src/Engine/System/RenderSystem.hpp

```diff
@@ -54,16 +56,22 @@ class RenderSystem {
 
     static void initFBO();
 
+    static void RenderSystem::CreateRenderBuffer(glm::ivec2 size);
```

Comment:
change to

static void CreateRenderBuffer(glm::ivec2 size);

for compatibility